webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -659,15 +660,14 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
                                     </td>
                                     <td>{{ frame.module }}</td>
                                     <td title="{{ frame.signature }}">{{ frame.signature }}</td>
-                                    {% if frame.source_link %}
                                     <td>
-                                        <a href="{{ frame.source_link }}">{{ frame.file }}:{{ frame.line }}</a>
-                                    </td>
-                                    {% else %}
-                                    <td>
-                                        {% if frame.file %} {{ frame.file }}:{{ frame.line }} {% endif %}
+                                        {% if frame.source_link %}
+                                            <a href="{{ frame.source_link }}">{{ frame.file }}:{{ frame.line }}</a>
+                                        {% else %}
+                                            {% if frame.file %}{{ frame.file }}:{{ frame.line }}{% endif %}
+                                        {% endif %}
                                     </td>
```

Comment:
I got tripped up that there was a `<td>` in each side of the `if` block here so I hoisted that part out and the `if` block only affects the contents now.